static/src/javascripts/projects/journalism/modules/audio-series-add-contributions.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+const renderContributionsBanner = el => {
+    const container = document.createElement('section');
```

Comment:
rather than constructing the html in js, can we load an html file like [here](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/journalism/modules/render-campaign.js#L4)?

Reply:
You know what, I might do that in a future because i need to amend this anyway to add Josh's new contrib banner design